freedomofpress/securedrop
# Tweaks for Tor nightly package testing

## Status

Ready for review

## Description of Changes

Contains code necessary to implement daily tor package tests against the tor nightly trusty repo. 

Subsequently also cleans up the ec2 logic and testinfra story surrounding the molecule `aws` scenario. Particularly, a developer no longer needs to specify a `vpc_id` but can utilize VPC names that default to FPF infra but can easily be over-ridden by developers testing against a personal EC2 environment.

Fixes #2536 

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?

Reviewer can test firing up an AWS environment locally if they'd like (not a requirement for the PR as far as I see it). You should check out the relevant jenkins jobs at https://ci.freedom.press/job/SD/job/tor-nightly-test/ though.

## Deployment

Nopeee - only affects CI


## Checklist

N/A
